clear/main.py

```diff
@@ -550,6 +550,28 @@ def route(prefix, vrf, namespace):
 helper = util_base.UtilHelper()
 helper.load_and_register_plugins(plugins, cli)
 
+# ("sonic-clear asic-sdk-health-event")
```

Comment:
Perhaps this should be "asic-sdk-health-events" instead of "asic-sdk-health-event" as this comments clears all asic sdk events.  

The HLD also uses "events". 
![image](https://github.com/sonic-net/sonic-utilities/assets/108555774/a48b8efc-d0cb-4738-ba3c-054abe9c3f60)

Reply:
Thanks. For now, I suggest keeping it as it is to avoid inconsistency.
I'll update HLD once all PRs have been merged.